src/cc2olx/models.py

```diff
@@ -124,9 +97,6 @@ def normalize(self):
         if not organization:
             return
         identifier = organization.get('identifier', 'org_1')
-        structure = organization.get('structure', 'rooted-hierarchy')
-        # An organization may have a title element.
-        title = None
```

Comment:
Same here, but I didn't even bother to parse it out, actually (I was roughly documenting with code as I read through the CC spec(s)).